### PR TITLE
Fix flighttrack name not updating in views

### LINKED ISSUE #1192: change of mscolab project does not update title on opened views
The Topview and other views keep the path of the initial used flightpath in the title.

e.g. starting with example switching to measurement keeps example in the title

also the text in TopView above the EPSG code is not changed.

### LINKED ISSUE #1209: Linear view vertical lines not resetting on flighttrack change
When changing a flight track, the linear view does not erase old vertical lines, as such it becomes quite convoluted and confusing.

### PR DESCRIPTION
**Pull request by Marilyth**
Fixes #1192 and fixes #1209
Changing the project/operation in the main UI now also changes the views window title and other references to it in the plot.

### CODE REVIEW

File: mslib/msui/mpl_qtwidget.py [hunk 1]
```diff
@@ -1030,6 +1030,12 @@ def redraw_xaxis(self):
                                                   lons[::tick_index_step])],
                                     rotation=25, horizontalalignment="right")
 
+            # Remove all vertical lines
```
Comment (joernu76) [inline mslib/msui/mpl_qtwidget.py:1038]: Who is drawing these lines? We should store the lines "properly" in a member to remove it cleanly...?

File: mslib/msui/viewwindows.py [hunk 2]
```diff
@@ -173,10 +178,20 @@ def setFlightTrackModel(self, model):
         """
         Set the QAbstractItemModel instance that the view displays.
         """
-        self.waypoints_model = model
+        super().setFlightTrackModel(model)
```
Comment (joernu76) [inline mslib/msui/viewwindows.py:180]: oh, python 3 super is much easier to use. Nice.

File: mslib/msui/viewwindows.py [hunk 3]
```diff
@@ -173,10 +178,20 @@ def setFlightTrackModel(self, model):
...
+
         if self.mpl is not None:
             self.mpl.canvas.set_waypoints_model(model)
 
+            # Update Top View flighttrack name
```
Comment (joernu76) [inline mslib/msui/viewwindows.py:185]: Same as above. I would prefer using members/methods for this instead of "parsing" the text. Not very maintainable.

**Review by joernu76 — CHANGES_REQUESTED** [review]

File: mslib/msui/viewwindows.py (hunk 3 above)
Comment (Marilyth) [reply mslib/msui/viewwindows.py:185]: Done

File: mslib/msui/mpl_qtwidget.py (hunk 1 above)
Comment (Marilyth) [reply mslib/msui/mpl_qtwidget.py:1038]: Done, also for Side View.

**Comment by ReimarBauer** [comment]
the operation name gets used and replaced by clicking on the map to the "internal name" Flight track. This is also added to the window title

namechangeonclick

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
By an operation the name changes  on e.g. inserting a waypoint to Flight track, see peek  comment.

**Review by joernu76 — DISMISSED** [review]

**Comment by Marilyth** [comment]
Should be fixed now

**Review by ReimarBauer — APPROVED** [review]
retried, works now. Thx.

**Pull request merged**